pkg/metrics/metrics.scss

```diff
@@ -523,32 +523,26 @@ html {
 
     // compressed minutes without events are simple bars
     .compressed {
-      display: grid;
-      grid-template-areas: ". utilization .";
-
-      --util-pct: calc(100% * var(--utilization));
-      grid-template-columns: calc(100% - var(--util-pct)) var(--util-pct);
+      --graph-parts: 1;
+      --util-pct: calc(100% / var(--graph-parts) * var(--utilization));
```

Comment:
That's not correct. No the --saturation value is ignored. Please leave it as is, I had a whole commit on top that adds the red part of the graph and it's conflicty. I want to solve this first.

Reply:
OK, we can revert this commit. It's probably worth looking into it though, de-grid-ifying where we can.